src/styles.css

```diff
@@ -82,44 +82,3 @@ footer {
     animation: pulsate 1.5s ease-out;
     animation-iteration-count: infinite;
 }
-
```

Comment:
⚠️ the js code is still applying these CSS classes! We don't have data to trigger the classes application. See https://github.com/assynour/bpmn-visu-tutorial-getting-started/blob/b0bbbb002d38cc90972ed9fd5cf0a071461706bd/src/index.js#L134-L138

Reply:
🤦‍♀️

Reply:
To simplify the visualization related to edges, can we agree on the following: 

- we store only the number of waiting instances (without referring to onTime, risky or critical categories)
- This number refers to instances that have been waiting for a while and that need intervention. 
- We diplay the number of these instances with an overlay and animate the edge (using red colors as we did for the critical category)